RPC config updates - Closes #1287 (#1317)
LiskHQ/lisk-docs: modules/ROOT/pages/advanced-explanations/communication.adoc

--
=====

=== IPC vs WS

> There are two different protocols each node can choose to enable the API:

Reviewer: ```suggestion
There are two ways through which a node can communicate via the API:
```
